src/core/css/inc/mixins.scss

```diff
@@ -70,23 +70,27 @@
   cursor: not-allowed;
 }
 
-/* stylelint-disable at-rule-empty-line-before, media-feature-no-missing-punctuation  */
+/*
+  stylelint-disable at-rule-empty-line-before,
+  media-feature-no-missing-punctuation
```

Comment:
does this still work when broken into a new line? Well, if the lint rules aren't failing I guess it does. Is the override still needed?